src/core/client/admin/routes/Community/UserRow.css

```diff
@@ -26,6 +26,9 @@
   font-weight: var(--font-weight-primary-semi-bold);
   font-size: var(--font-size-2);
   line-height: 1.14;
+  line-break: anywhere;
```

Comment:
```suggestion
```

Reply:
Remove this, as it is too aggressive: 
![image](https://user-images.githubusercontent.com/14221600/97626878-d20dcb00-1a2a-11eb-83fa-5e36d5afbc8a.png)